test/core.js

```diff
@@ -14,6 +14,8 @@ const windows = process.platform === 'win32'
 const veryOldNode = semver.lt(process.versions.node, '16.0.0')
 const oldNode = semver.lt(process.versions.node, '18.0.0')
 
+const execute = (/** @type {() => void} */ f) => f()
```

Comment:
Maybe a comment above this to say that it will be changed by the build script to an async version?

Reply:
Also: I somewhat prefer the term `run`: a) it’s shorter, b) it’s not a synonym for killing things. What do you think?

Reply:
I'm less inclined to leave a comment about the function being changed to async because there's a lot of sync/async transformations going on in this file. I would prefer adding a comment in the build script of why we're doing this.

Reply:
Added a comment in `script.js`